test/test.js

```diff
@@ -4,44 +4,11 @@ var url = require('url');
 var expect = require('chai').expect;
 var objectAssign = require('object-assign');
 var format = require('string-format');
-var rimraf = require("rimraf");
-var fs = require('fs');
 var common = require('../lib/common')
 
-/*rimraf('sitespeed-result', function(err) {
-    console.log("delete legacy test data success")
-});
-*/
-
-var path1 = __dirname + '/../../../config/budget-testcases.json';
-var path2 = __dirname + '/../config/budget-testcases.json';
-var config = process.env.BUDGET_TESTCASES;
-if (!config) {
-    if (fs.existsSync(path1)) {
-        console.log("test case config path is " + path1)
-        try {
-            config = require(path1);
-        } catch (e) {
-            console.dir(e);
-            console.error('Error loading budget test case file ' + path1);
-            process.exit(1);
-        }
-    } else if (fs.existsSync(path2)) {
-        console.log("test case config path is " + path2)
-        try {
-            config = require(path2);
-        } catch (e) {
-            console.dir(e);
-            console.error('Error loading budget test case file ' + path2);
-            process.exit(1);
-        }
-    } else {
-        console.error('please configure test case file in ' + path1 + ' or ' + path2);
-        process.exit(1);
-    }
-}
-var baseUrl = config.baseUrl;
-var testCases = config.testCases;
+var baseUrl = spUtilsCfg.baseUrl;
```

Comment:
@kakazuo So, per my https://github.com/aerwin/sitespeed-utils/pull/4/commits/a75afe394fce80d16d17987d09f2d3f9ae89d0a5#r61295134, I guess you must be relying on `spUtilsCfg` being a global variable? Isn't there some way to pass it in when you create the `Mocha` object?